integration/integrate-using-email.adoc

```diff
@@ -0,0 +1,17 @@
+[id="integrate-using-email"]
+= Integrating with email
+include::modules/common-attributes.adoc[]
+:context: integrate-using-email
+
+toc::[]
+
+[role="_abstract"]
+You can use email as a notification method, and you can forward alerts from the {product-title} to a standard email provider. This topic explains how to integrate the {product-title-short} platform with your email provider.
+
+To forward alerts from the {product-title-short} platform to an email address, you can either use:
+
+* Default recipients
+* Deployment annotations
+
+include::modules/configure-acs-kube-security-platform.adoc[leveloffset=+1]
```

Comment:
**issue:** Can you please fix this?

It appears as unresolved directive in the preview:
>Unresolved directive in modules/configure-acs-kube-security-platform.adoc - include::modules/common-attributes.adoc[] :context: configure-acs-kube-security-platform